html/semantics/embedded-content/the-iframe-element/srcdoc_process_attributes.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Whenever `srcdoc` attribute is set, changed, or removed, the UA must process the &lt;iframe> attributes</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element:process-the-iframe-attributes-2">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+function createIFrameWithBlobSrc() {
+  var iframe = document.createElement("iframe");
+  iframe.src = URL.createObjectURL(new Blob(["src"], {type: "text/html"}));
+  return iframe;
+}
+
+async_test(function(t) {
+  var iframe = createIFrameWithBlobSrc();
+  iframe.onload = t.step_func(function() {
+    assert_equals(iframe.contentDocument.location.protocol, "blob:");
+    assert_equals(iframe.contentDocument.body.textContent, "src");
+
+    iframe.onload = t.step_func(function() {
+      assert_false(isSetting);
+      assert_equals(iframe.contentDocument.location.href, "about:srcdoc");
+      assert_equals(iframe.contentDocument.body.textContent, "srcdoc");
+      t.done();
+    });
+
+    var isSetting = true;
+    iframe.setAttribute("srcdoc", "srcdoc");
+    isSetting = false;
+  });
+
+  document.body.appendChild(iframe);
+}, "Setting `srcdoc` triggers attributes processing");
+
+async_test(function(t) {
+  var iframe = createIFrameWithBlobSrc();
+  iframe.srcdoc = "old";
+  iframe.onload = t.step_func(function() {
+    assert_equals(iframe.contentDocument.location.href, "about:srcdoc");
+    assert_equals(iframe.contentDocument.body.textContent, "old");
+
+    iframe.onload = t.step_func(function() {
+      assert_false(isChanging);
+      assert_equals(iframe.contentDocument.location.href, "about:srcdoc");
+      assert_equals(iframe.contentDocument.body.textContent, "new");
+      t.done();
+    });
+
+    var isChanging = true;
+    iframe.srcdoc = "new";
+    isChanging = false;
+  });
+
+  document.body.appendChild(iframe);
+}, "Changing `srcdoc` triggers attributes processing");
```

Comment:
Maybe "Setting srcdoc property ..."?

Reply:
I've slightly improved test cases description in b764ed9, but kept "changing": attributes should not be processed after `iframe.srcdoc = iframe.srcdoc`, right?

Reply:
Per the DOM Standard that is a change.

Reply:
Thank you, resolved in https://github.com/web-platform-tests/wpt/pull/20084/commits/522f4c5bddb65c56b4440cf0cd0ed186edb37235.